src/main/perl/BlockdevFactory.pm

```diff
@@ -7,27 +7,60 @@ NCM::BlockdevFactory
 =cut
 
 use NCM::Blockdevices qw ($reporter);
-use NCM::MD;
-use NCM::LVM;
-use NCM::LV;
-use NCM::Disk;
-use NCM::Partition;
-use NCM::File;
-use NCM::Tmpfs;
-use NCM::VXVM;
-
+use Module::Load;
 use CAF::Process;
-
-use constant BASEPATH => "/system/blockdevices/";
-use constant PARTED	=> qw (/sbin/parted -s --);
-use constant PARTEDEXTRA => qw (u MiB);
-use constant PARTEDP	=> 'print';
+use Readonly;
+
+Readonly my $BASEPATH => "/system/blockdevices";
+Readonly my @PARTED	=> qw (/sbin/parted -s --);
+Readonly my @PARTEDEXTRA => qw (u MiB);
+Readonly my $PARTEDPRINT => 'print';
+# if all is put in qw(), perl wans with 'Possible attempt to separate words with commas'
+Readonly my $DMINFO => [qw(dmsetup info -C --noheadings --separator : -o), 'name,subsystem'];
+
+Readonly my %BUILD_MAP => (
```

Comment:
This seems to control which modules are loaded for a given PAN path. Can this be documented explicitly to make it easy to see how to add new types?